provider/cluster/kube/builder.go

```diff
@@ -243,6 +338,231 @@ func (b *serviceBuilder) ports() []corev1.ServicePort {
+					},
+				},
+				PolicyTypes: []netv1.PolicyType{
+					netv1.PolicyTypeEgress,
+				},
+			},
+		},
+
+		{
+			// Allow egress between services within the namespace.
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   netPolEgressInternalAllow,
+				Labels: b.labels(),
+			},
+			Spec: netv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						akashNetworkNamespace: lidNS(b.lid),
+					},
+				},
+				Ingress: []netv1.NetworkPolicyIngressRule{
+					{ // Allow Network Connections from same Namespace
+						From: []netv1.NetworkPolicyPeer{
+							{
+								NamespaceSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{
+										akashNetworkNamespace: lidNS(b.lid),
+									},
+								},
+							},
+						},
+					},
+				},
+				PolicyTypes: []netv1.PolicyType{
+					netv1.PolicyTypeEgress,
+				},
+			},
+		},
+
+		{ // Allow egress to all IPs, EXCEPT local cluster.
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   netPolEgressAllowExternalCidr,
+				Labels: b.labels(),
+			},
+			Spec: netv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						akashNetworkNamespace: lidNS(b.lid),
+					},
+				},
+				Egress: []netv1.NetworkPolicyEgressRule{
+					{ // Allow Network Connections to Internet, block access to internal IPs
+						To: []netv1.NetworkPolicyPeer{
+							{
+								IPBlock: &netv1.IPBlock{
+									CIDR: "0.0.0.0/0",
+									Except: []string{
+										// TODO: Full validation and correction required.
+										// Initial testing indicates that this exception is being ignored;
+										// eg: Internal k8s API is accessible from containers, but
+										// open Internet is made accessible by rule.
+										"10.0.0.0/8",
+									},
+								},
+							},
+						},
+					},
+				},
+				PolicyTypes: []netv1.PolicyType{
+					netv1.PolicyTypeEgress,
+				},
+			},
+		},
+
+		{
+			// Allow egress to Kubernetes internal subnet for DNS
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   netPolEgressAllowKubeDNS,
+				Labels: b.labels(),
+			},
+			Spec: netv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						akashNetworkNamespace: lidNS(b.lid),
+					},
+				},
+				Egress: []netv1.NetworkPolicyEgressRule{
+					{ // Allow Network Connections from same Namespace
+						Ports: []netv1.NetworkPolicyPort{
+							{
+								Protocol: &dnsProtocol,
+								Port:     &dnsPort,
+							},
+						},
+						To: []netv1.NetworkPolicyPeer{
+							{
+								IPBlock: &netv1.IPBlock{
+									CIDR: "10.0.0.0/8",
+								},
+							},
+						},
+					},
+				},
+				PolicyTypes: []netv1.PolicyType{
+					netv1.PolicyTypeEgress,
+				},
+			},
+		},
+	}, nil
+}
+
+// Update a single NetworkPolicy with correct labels.
+func (b *netPolBuilder) update(obj *netv1.NetworkPolicy) (*netv1.NetworkPolicy, error) { // nolint:golint,unparam
```

Comment:
is the nolint still needed?  looks like all params are used.

Reply:
It's required because the methods always return nil to implement the interface.